Source/OctaneGUI/Controls/TextInput.cpp

```diff
@@ -890,6 +891,11 @@ void TextInput::AddText(const std::u32string& Contents)
     {
         return;
     }
+    
+    if (m_Text->Length() + Contents.length() > m_MaxCharacters)
+    {
+        return;
+    }
```

Comment:
The contents of the incoming string should be stripped to allow the control to meet the m_MaxCharacters limit e.g. the m_MaxCharacters = 2 and the incoming string is "Hello", the control should have the text "He".

Reply:
Moved within https://github.com/thnom/OctaneGUI/commit/079fb6b516a88c2a4334a7b7518609cf522d5bb9 to use _Stripped_ version of.